list/styles2-lists.css

```diff
@@ -0,0 +1,78 @@
+/* Lists Main */
+.lists-container {
+  margin: 3rem;
+}
+
+.lists-main-heading {
+  color: yellow;
+}
+.lists-snippet iframe {
+  margin: 1rem 0rem;
+  width: 524px;
```

Comment:
never use px greater than 10px for widths, paddings, font-sizes

Reply:
I'll keep in mind from the next time